django/core/management/templates.py

```diff
@@ -76,12 +75,8 @@ def handle(self, app_or_project, name, target=None, **options):
             top_dir = path.join(os.getcwd(), name)
             try:
                 os.makedirs(top_dir)
-            except OSError as e:
-                if e.errno == errno.EEXIST:
-                    message = "'%s' already exists" % top_dir
-                else:
-                    message = e
-                raise CommandError(message)
+            except FileExistsError:
+                raise CommandError("'%s' already exists" % top_dir)
```

Comment:
IIRC raising CommandError prevents management commands from displaying a stack trace. This doesn't seem very important but I wanted to point out the change in behavior in case it was accidental.